README.rst

```diff
@@ -1,23 +1,178 @@
+------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+Make sure you've installed:
+
+-  Python 2.7.x development environment (virtualenv strongly
+   recommended)
+
+   *Hint*: See `runtime.txt`_ for the exact version
+-  `Heroku Command Line`_
+
+   All ``heroku`` commands can be performed through the Heroku web-based
+   dashboard as well, if you don't want to use the CLI.
+
+Set up
+~~~~~~
+
+1. Log in using the email address and password you used when creating
+   your Heroku account:
+
+   .. code:: sh
+
+       heroku login
+
+   Authenticating is required to allow both the ``heroku`` and ``git``
+   commands to operate.
+
+2. Add the Heroku app repo as a git remote:
+
+   .. code:: sh
+
+       heroku git:remote -a openedx-webhooks-staging
+
+3. Verify that the remote is added properly:
+
+   .. code:: sh
+
+       git remote -v
+
+   You should see output similar to:
+
+   .. code:: sh
+
+       heroku  https://git.heroku.com/openedx-webhooks-staging.git (push)
+       heroku  https://git.heroku.com/openedx-webhooks-staging.git (fetch)
+       origin  git@github.com:edx/openedx-webhooks.git (fetch)
+       origin  git@github.com:edx/openedx-webhooks.git (push)
+
+Develop
+-------
+
+This app relies on the following addons from Heroku:
+
+-  PostgreSQL
+-  Redis
+-  IronMQ
+-  Papertrail
+-  Scheduler
+
+While it's possible to replicate the entire stack locally, it'll be
+difficult to ensure consistent experience for each developer. Instead,
+we utilize the `pipeline`_ facility offered by Heroku to handle our
+development needs.
+
+The general development cycle is:
+
+Code → Deploy branch to staging → Test → Iterate
+
+To deploy a local branch to staging:
+
+.. code:: sh
+
+    git push heroku [branch_or_tag_or_hash:]master
+
+Once you're satisfied with your changes, go ahead and open a pull
+request per normal development procedures.
+
+Smoke test the deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Navigate to https://openedx-webhooks-staging.herokuapp.com to make sure
+the app has started. If the URL is too hard to remember, you can also
+use:
+
+.. code:: sh
+
+    heroku open
+
+Run Tests
+---------
 
 .. code:: sh
 
-    pip install -r dev-requirements.txt
+    make install-requirements
     make test
 
+Deploy
+------
+
+In most cases, you'll want to deploy by promoting from staging to
+production.
+
+**Prior to the promotion**, make sure all the changes have been merged
+to ``master``, and you've deployed the ``master`` branch successfully to
+staging.
+
+The general workflow is:
+
+Merge to ``master`` → Deploy ``master`` to staging → Test → Promote to
+production
+
+When you're ready to promote from staging to production:
+
+.. code:: sh
+
+    heroku pipelines:promote -r heroku
+
+Ensure the same versions are deployed
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Make sure the same git commit is deployed to both environments. First
+see what's deployed on staging:
+
+.. code:: sh
+
+    heroku releases -n 1
```

Comment:
Yes. I believe most or all `heroku` commands look for the app referenced via `heroku` git remote as default. The reason it didn't work for you is probably because you didn't have view permissions to the `openedx-webhooks-staging` app.